lib/joi/joi.bundle.js

```diff
@@ -82,7 +82,7 @@ module.exports =
 /******/
```

Comment:
Generated according to the instructions in https://github.com/OldSneerJaw/couchster/blob/v1.2.0/lib/joi/README.md